### PR TITLE
Add templates for managed disk ADE on Linux

### PR DESCRIPTION
**Pull request by krkhan**
- Add 101-vm-full-disk-encrypted-rhel-md (cloned from native disk counterpart 101-vm-full-disk-encrypted-rhel)
- Add 101-vm-simple-rhel-md (cloned from native disk counterpart  101-vm-simple-rhel)

### CODE REVIEW

File: 101-vm-full-disk-encrypted-rhel-md/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        "templateLink": {
+          "contentVersion": "1.0.0.0",
+          "uri": "[variables('createVmUrl')]"
+        }
+      }
+    },
+    {
+      "type": "Microsoft.Compute/virtualMachines/extensions",
+      "name": "[concat(parameters('vmName'), '/', variables('setupRaidExtensionName'))]",
+      "apiVersion": "2015-06-15",
```
Comment (SudhakaraReddyEvuri) [inline 101-vm-full-disk-encrypted-rhel-md/azuredeploy.json]: Use the preview CRP API version even for extensions

Comment (krkhan) [inline 101-vm-full-disk-encrypted-rhel-md/azuredeploy.json]: Addressed with commit 657267debf2af7c74d13f5cde50df2cdaffb3f52

File: 201-encrypt-running-linux-vm/azuredeploy.json [hunk 2]
```diff
@@ -90,7 +90,7 @@
     },
     "_artifactsLocation": {
       "type": "string",
-      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master",
+      "defaultValue": "https://raw.githubusercontent.com/krkhan/azure-quickstart-templates/master",
```
Comment (singhkays) [inline 201-encrypt-running-linux-vm/azuredeploy.json]: Can you change all the URLs to Azure?

Comment (krkhan) [inline 201-encrypt-running-linux-vm/azuredeploy.json]: Fixed with 4f915ef70efd7ad3a4f8b7bcd50501b2083e797d

**Comment by singhkays** [comment]
lgtm, merged

**Pull request merged**